lib/timeutils/tests/test_scan-timestamp.c

```diff
@@ -287,10 +289,6 @@ Test(parse_timestamp, cisco_timestamps)
 
 Test(parse_timestamp, rfc5424_timestamps)
 {
-  /* the timestamp '-' means the current time */
-  _expect_rfc5424_timestamp_eq("-", "2017-12-13T09:10:23.123+01:00");
```

Comment:
As I see this functionality is still available, just pulled up into `log_msg_parse_timestamp`. Would it make sense to add a testcase for that to maintain coverage?

Reply:
I've pushed another patch on top, which adds a testcase to test_msgparse that covers the "-"
timestamp.